docs/analyzer/index.md

```diff
@@ -2,27 +2,48 @@
 
 ## Description
 
+The Presidio analyzer is a Python based service for detecting PII entities in text.
+
+During analysis, it runs a set of different *PII Recognizers*,
+each one in charge of detecting one or more PII entities using different mechanisms.
+
+Presidio analyzer comes with a set of predefined recognizers,
+but can easily be extended with other types of custom recognizers.
+Predefined and custom recognizers leverage regex, Named Entity Recognition and other types of logic to detect PII in unstructured text.
+  
 ## Table of contents
 
+- [Installation](#installation)
+- [Getting started](#getting-started)
+  - [Running in Python](#running-in-python)
+  - [Running using Docker](#running-using-docker)
+- [Creating PII recognizers](#creating-pii-recognizers)
+- [API reference](#api-reference)
+
 ## Installation
 
 ## Getting started
 
-- Running in Python.
-- Running using Docker.
-- Creating PII recognizers.
-- [Supporting additional languages and ML models](languages.md)
-- [Outputing the analyzer decision process](decision_process.md)
-- API reference.
-
 ### Running in Python
 
 ### Running using Docker
 
-### Creating PII recognizers
+### Additional languages
+
+See [supporting additional languages and ML models](languages.md).
+
+### Explaining the presidio-analyzer decision process
+
+See [tracking the analyzer decision process](decision_process.md).
+
+## Creating PII recognizers
+
+### How to add a new recognizer
+
+TODO AZDO 2856: copy from analyzer README
```

Comment:
What is the purpose of this TODO? another option would be to leave it blank and add an issue (which already seems to exist in AZDO 2856 if it will be fixed shortly anyways.

Reply:
We leave some TODOs just to verify that we didn't forget updating all places. Before merging back to the main branch, we would scan to make sure we didn't forget any TODOs.